core/src/main/java/io/github/tcdl/msb/monitor/aggregator/DefaultChannelMonitorAggregator.java

```diff
@@ -71,31 +74,52 @@ void onHeartbeatResponses(List<Message> heartbeatResponses) {
         for (Message msg : heartbeatResponses) {
             aggregateInfo(aggregatorStats, msg);
```

Comment:
You ignore the return value of aggregateInfo here. Looks fishy.
If you're unable to process one heartbeat response then you aggregatorStats might not be accurate but you still call the handler. 

It might be OK but I would argue that at least logging should be similar to "onAnnounce". There you log error message when payload is not valid so why don't you doing it here? In fact I wouldn't log anything in both cases because "aggregateInfo" already have a very detailed logging that explains clearly why the message is considered to be invalid